liczek/src/main/java/pl/fratik/liczek/commands/LiczekCommand.java

```diff
@@ -109,4 +124,10 @@ public boolean execute(@NotNull @Nonnull CommandContext context) {
         return false;
     }
 
+    private String hasPermission(CommandContext ctx, GuildChannel channel) {
```

Comment:
miałem na myśli "użyj tego standardowego tekstu kiedy bot nie ma uprawnień na embedy" ale ok, tak też może być

natomiast jak już robisz embeda to dodaj tego perma w constructorze

Reply:
się robi